examples/android/WORKSPACE

```diff
@@ -0,0 +1,71 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+RULES_JVM_EXTERNAL_TAG = "2.8"
+
+RULES_JVM_EXTERNAL_SHA = "79c9850690d7614ecdb72d68394f994fef7534b292c4867ce5e7dec0aa7bdfad"
+
+ANDROID_API = 28
+
+ANDROID_TOOLS_VERSION = "28.0.2"
```

Comment:
let's not pin this. the CI may not always have 28.0.2.

Reply:
Done.